docs/sdks/swift.md

```diff
@@ -0,0 +1,483 @@
+---
+title: Swift SDK
+description: Learn how to use Featurevisor Swift SDK
+ogImage: /img/og/docs-sdks-swift.png
+---
+
+## Installation
```

Comment:
worth adding a link to the featurevisor-swift repo from this page as well.

we can say that more API docs can be found there or something. will let you decide how to position it.